Library/ViewModels/RewardAddOnCardViewModel.swift

```diff
@@ -103,8 +103,11 @@ public final class RewardAddOnCardViewModel: RewardAddOnCardViewModelType, Rewar
       .map(itemsLabelAttributedText)
       .skipNil()
 
+    // TODO: Update string with translations [mbl-1667](https://kickstarter.atlassian.net/browse/MBL-1667)
     self.estimatedShippingLabelText = Signal.combineLatest(reward, shippingRule)
-      .map { reward, shippingRule in estimatedShippingText(for: reward, selectedShippingRule: shippingRule) }
+      .map { reward, shippingRule in
+        "About \(estimatedShippingText(for: reward, selectedShippingRule: shippingRule))"
```

Comment:
Do you know if ranges are represented by `min - max` in all the languages we translate into? Specifically, is that true for Japanese? If not, we should have two separate translated strings; one "About {value}" and a separate "About {min_value} - {max_value}".

Reply:
This is and interesting point. I'm reaching out to verify

Reply:
If we need to update our strings, I should do that in a separate PR. I've reached out to design and the team so that the conversation can start ASAP.